components/bl-autocomplete/src/helpers/value-validation.js

```diff
@@ -0,0 +1,17 @@
+export function valueValidation(value, options, hasGroup) {
```

Comment:
valueValidation -> validateValue/valueValidator

same for options

And make it in one file.

Also it could be a map

Validators.options(...)
Validators.value(...)